### ISSUE TITLE
Models w/Contrastive Learning Objective

### ISSUE THREAD

**Issue by PrithivirajDamodaran** [UKPLab/sentence-transformers#1202]
Please point out the models which are fine-tuned using contrastive learning objective as opposed to cross-entropy objective under the sentence transformer models. 

Thanks in Advance

**Comment by nreimers** [comment]
None of the uploaded models have been trained with contrastive learning, as it performs rather poorly compared  to MultipleNegativesRankingLoss

https://www.sbert.net/docs/package_reference/losses.html#contrastiveloss
You have pairs and a label, that indicate if the pair is positive (and should be close in vector space) or if it is negative (and should be far away in vector space)

**Comment by nreimers** [comment]
The newer models have a Train_script.py in the git of the model hub uploaded, which is the code that was used to train the respective model
